src/bandersnatch/tests/plugins/test_storage_plugins.py

```diff
@@ -432,20 +438,6 @@ def setUp_plugin(self) -> None:
             )
         )
 
-    def setUp_mirrorDirs(self) -> None:
-        pypi_dir = (
-            "srv/pypi" if self.container is None else f"{self.container}/srv/pypi"
-        )
-        assert self.tempdir
-        self.mirror_base_path = os.path.join(self.tempdir.name, pypi_dir)
-        self.web_base_path = os.path.join(self.mirror_base_path, "web")
-        self.json_base_path = os.path.join(self.web_base_path, "json")
-        self.pypi_base_path = os.path.join(self.web_base_path, "pypi")
-        self.simple_base_path = os.path.join(self.web_base_path, "simple")
-        os.makedirs(self.json_base_path, exist_ok=True)
-        os.makedirs(self.pypi_base_path, exist_ok=True)
-        os.makedirs(self.simple_base_path, exist_ok=True)
```

Comment:
What sets these up now?

Reply:
I was so puzzled that this method is not actually used in the entire file, so I deleted it.